redis-objects.gemspec

```diff
@@ -28,7 +28,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "connection_pool"
 
   # Compatibility testing
-  spec.add_development_dependency "redis-namespace"
```

Comment:
Let's use the gem from rubygems.org instead of git
The 1.6 version is released and can use `redis` 4 (thanks to you @ota42y さん https://github.com/resque/redis-namespace/pull/136)

```
spec.add_development_dependency "redis-namespace", "~> 1.6"
```

Reply:
I fixed.
But I don't set `"~> 1.6"` because this gem work well 1.5.
So the user using 1.5 can upgrade redis-object without upgrading redis-namespace :)